### PR TITLE
chore(docs): Some improvements to v1 spec docs

### PR DESCRIPTION
**Pull request by hermanschaaf**

### CODE REVIEW

File: website/pages/docs/reference/destination_spec.md [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+Name of the plugin. If you have multiple destination plugins this should be unique.
 
-`version` (`string`, `required`) - Should be in the form of `vX.Y.Z` (or `vX.Y.Z-sometag` as per [semver](https://semver.org/)). Latest versions can be generate by `cloudquery gen` command. Also you can find all official plugins versions under [cloudquery/cloudquery/releases](https://github.com/cloudquery/cloudquery/releases) and for community you find in the relevant community repository.
+### path
+(`string`, `optional`)
 
-`registry` (`string`, `optional`, default: `github`, available: `github,local,grpc`) - `github`: cloudquery will look and download the plugin from github and then execute it. `local`: cloudquery will execute the plugin from a local apth. `grpc`: mostly useful in debug mode when plugin is already running in a different terminal, cloudquery will connect to the gRPC plugin server directly without spawning the process.
+By default, `path` is automatically inferred from `name`. For example for `postgresql`, `path` will be `cloudquery/postgresql`, or for `community_org/dest_name` `path` will be `community_org/dest_name`.
```
Comment (erezrokah) [inline website/pages/docs/reference/destination_spec.md:15]:
```suggestion
Configures how to retrieve the plugin. For plugins hosted on GitHub, `path` is inferred from `name` by default.
For example `name: postgresql` will resolve `path` to `clouduquery/postgresql`. If you'd like to use a plugin that is not hosted on the CloudQuery repository, provider the full path to the repository such as `community-github-org/community-github-repo`
```

File: website/pages/docs/reference/destination_spec.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+
+### version
+(`string`, `required`)
+
+`version` should be in the form of `vX.Y.Z` (or `vX.Y.Z-sometag` as per [semver](https://semver.org/)). Latest versions can be generated using `cloudquery gen` command. You can find all official plugin versions under [cloudquery/cloudquery/releases](https://github.com/cloudquery/cloudquery/releases), and for community plugins you can find it in the relevant community repository.
```
Comment (erezrokah) [inline website/pages/docs/reference/destination_spec.md:20]:
```suggestion
`version` must be a valid [SemVer](https://semver.org/)), e.g. `vMajor.Minor.Patch`. Latest versions can be generated using `cloudquery gen` command. You can find all official plugin versions under [cloudquery/cloudquery/releases](https://github.com/cloudquery/cloudquery/releases), and for community plugins you can find it in the relevant community repository.
```

**Review by erezrokah — APPROVED** [review]
Added a few non blocking comments for the destination (that can be applied to source too).

Looks good 🚀

**Comment by hermanschaaf** [comment]
@yevgenypats PTAL when you get a chance; this also adds some notes about the upcoming `overwrite-delete-stale` mode

**Pull request merged**